simpleflow/swf/executor.py

```diff
@@ -176,7 +176,7 @@ def __init__(self, domain, workflow_class, task_list=None, repair_with=None,
                  ):
         super(Executor, self).__init__(workflow_class)
         self._history = None
-        self._execution_context = {}
+        self._replay_context = {}
```

Comment:
Sometimes it's "running context" sometimes it's "replay context" ? If it's the same underlying concept, I'd go with unified wording. "running context" or "run context" or "replay context" are all fine by me.

(I like this first commit very much tho, thanks a lot!)

Reply:
Yeah, `run_context` wins IMO. "Replay context" is meaningful for SWF only